henson_amqp/__init__.py

```diff
@@ -269,8 +269,9 @@ def send(self, message, *, routing_key=None):
                 be used. Defaults to ``None``.
         """
         properties = {
-            'delivery_mode': self.app.settings['AMQP_DELIVERY_MODE'].value,
+            'delivery_mode': self.app.settings['AMQP_DELIVERY_MODE'],
```

Comment:
Shouldn’t there be something to convert the Enum into an int if that come is.
`DeliveryMode.NONPERSISTENT` - > 1 etc
Like a check to see if it is a string and try to get the value from that

Reply:
For backwards compatibility

Reply:
From above:

>As it turns out, aioamqp does not need the explicitly conversion of the IntEnum -> int via the .value property. You can just pass in the IntEnum and it handles it under the hood.